src/containers/BankApi/BankApiInvalidBlocks/index.tsx

```diff
@@ -40,7 +40,7 @@ const BankApiInvalidBlocks: FC = () => {
       <RequestResponseSnippet
         code={`{
   "count": 234,
-  "next": "http://143.110.137.54/invalid_blocks?limit=50&offset=50",
+  "next": "http://143.110.137.54/invalid_blocks?limit=50&offset=2",
```

Comment:
Fix the count as well and if you want to change the arguments then it should be
limit=2
offset=2
for 'next' attribute.

Reply:
Count does not change, it is the total number of element (not only the one displayed here)
+1 for the `next`